### PR TITLE
Update BOTMETA for VMware maintainers

### PR DESCRIPTION
**Pull request by mtnbikenc**
##### SUMMARY

@jcpowermac and @mtnbikenc are no longer actively maintaining VMware modules and would like the bot to stop sending notifications for new/updated modules.

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

.github/BOTMETA.yml

##### ANSIBLE VERSION

```
devel
```

### CODE REVIEW

**Comment by jcpowermac** [comment]
LGTM

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -297,35 +297,37 @@ files:
...
+  $modules/cloud/vmware/vmware_datacenter.py: $team_ansible kamsz
+  $modules/cloud/vmware/vmware_dns_config.py: $team_ansible
+  $modules/cloud/vmware/vmware_dvs_host.py: $team_ansible
+  $modules/cloud/vmware/vmware_dvs_portgroup.py: $team_ansible
+  $modules/cloud/vmware/vmware_dvswitch.py: $team_ansible
   $modules/cloud/vmware/vmware_guest.py: dagwieers
   $modules/cloud/vmware/vmware_guest_facts.py: nerzhul
-  $modules/cloud/vmware/vmware_guest_find.py: $team_vmware
-  $modules/cloud/vmware/vmware_host.py: $team_ansible jcpowermac dagwieers
+  $modules/cloud/vmware/vmware_guest_find.py:
```
Comment (samdoran) [inline .github/BOTMETA.yml:315]: I believe `$team_vmware` should still be responsible for this module.

Comment (mtnbikenc) [reply .github/BOTMETA.yml:315]: `$team_vmware` already applied to the entire `vmware/` directory and has been moved to `maintainers:` on line 301.

Comment (samdoran) [reply .github/BOTMETA.yml:315]: Thanks for the clarification.

**Pull request merged**